### PR TITLE
Updated gitignore files

### PR DESCRIPTION
**Pull request by jesper-friis**
# Description
Cleaned up .gitignore files.

## Type of change
- [x] Bug fix & code cleanup
- [ ] New feature
- [ ] Documentation update
- [ ] Test update

## Checklist for the reviewer
This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

File: python/.gitignore [hunk 1]
```diff
@@ -1,44 +1,6 @@
...
-
-# Generated build docker files
-Dockerfile-*linux*[^.template]
-
-# Units (pint) files
-units.cpython-*.pyc
-bindings/python/triplestore/__pycache__
-
-# Documentation
-pydoc/autoapi/
```
Comment (francescalb) [inline python/.gitignore:41]: Why remove this?

**Comment by francescalb** [comment]
Is the reason for this clean-up is to move gitignore to the parent directory? 

Also, there are quite a few thigns that are not only moved but removed, why?

File: python/.gitignore (hunk 1 above)
Comment (jesper-friis) [reply python/.gitignore:41]: They are included in the top-level gitignore file.

**Review by francescalb — APPROVED** [review]

**Comment by jesper-friis** [comment]
> Is the reason for this clean-up is to move gitignore to the parent directory?
> 
> Also, there are quite a few thigns that are not only moved but removed, why?

Most of what is removed was an unnecessary duplication of the toplevel .gitignore file. There were also some too wide entries that matched too much.

**Pull request merged**